evil-collection-company.el

```diff
@@ -54,24 +54,26 @@ be set through custom or before evil-collection loads."
 
 (defun evil-collection-company-setup ()
   "Set up `evil' bindings for `company'."
-  (define-key company-active-map (kbd "C-n") 'company-select-next-or-abort)
-  (define-key company-active-map (kbd "C-p") 'company-select-previous-or-abort)
-  (define-key company-active-map (kbd "C-j") 'company-select-next-or-abort)
-  (define-key company-active-map (kbd "C-k") 'company-select-previous-or-abort)
-  (define-key company-active-map (kbd "M-j") 'company-select-next)
-  (define-key company-active-map (kbd "M-k") 'company-select-previous)
+  (evil-collection-define-key nil 'company-active-map
```

Comment:
I'm curious if this keeps the same keymap precedence before. I think I used define-key because other keys were still taking over, over evil keymaps.

Reply:
More specifically, `evil-define-key*` with `nil` as the state is equivalent to using a bunch of `define-key` calls like you had here. `evil-collection-define-key` adds the new filtering and reporting functionality on top.